packages/pow/src/pow.ts

```diff
@@ -0,0 +1,237 @@
+import { defaultHost } from '@textile/context'
+import { Identity } from '@textile/crypto'
+import { GrpcAuthentication } from '@textile/grpc-authentication'
+import {
+  AddrsResponse,
+  InfoResponse,
+  ListDealRecordsConfig,
+  ListRetrievalDealRecordsResponse,
+  ListStorageDealRecordsResponse,
+  NewAddrResponse,
+  SendFilResponse,
+  ShowAllResponse,
+} from '@textile/grpc-powergate-client/dist/ffs/rpc/rpc_pb'
+import { CheckResponse } from '@textile/grpc-powergate-client/dist/health/rpc/rpc_pb'
+import {
+  ConnectednessResponse,
+  FindPeerResponse,
+  PeersResponse,
+} from '@textile/grpc-powergate-client/dist/net/rpc/rpc_pb'
+import { BalanceResponse } from '@textile/grpc-powergate-client/dist/wallet/rpc/rpc_pb'
+import { KeyInfo, UserAuth } from '@textile/security'
+import log from 'loglevel'
+import {
+  addrs,
+  balance,
+  connectedness,
+  findPeer,
+  health,
+  info,
+  listRetrievalDealRecords,
+  listStorageDealRecords,
+  newAddr,
+  peers,
+  sendFil,
+  show,
+  showAll,
+} from './api'
+
+const logger = log.getLogger('users')
+
+export class Pow extends GrpcAuthentication {
```

Comment:
I think at minimum all the public methods need a doc string. ideally at least 1 or 2 examples for the top level Pow class for now (i'll open a ticket for creating an example for the other public methods later)

You can see an example of what this looks like here,

https://textileio.github.io/js-hub/docs/hub.buckets

The Buckets class is documented with 3 examples, those feel like an example or a few good minimum to give someone an idea of how to get started. 

The table below, "Methods" shows the list of all public methods with the one line description of the method, that's why i think those one-liners are a minimum for each of the new methods

Reply:
Ok got the descriptions and params in for the public methods. Will address https://github.com/textileio/js-hub/issues/489 later.